docs/source/fundamentals/interfaces.rst

```diff
@@ -3,6 +3,12 @@
 Interfaces
 ==========
 
+.. note::
+
+    If you encounter issues you cannot resolve, simply ask in our
+    `Slack community <https://join.slack.com/t/lineacommunity/shared_invite/zt-18kizfn3b-1Qu_HDT3ahGudnAwoFAw9Q>`_'s
+    ``#support`` channel. We are always happy and ready to help you!
+
```

Comment:
This and other snippets are used quite a bit in different sections. recommend making this into a template and including that in all the places. It will be more maintainable that way.

Reply:
Great point, done!